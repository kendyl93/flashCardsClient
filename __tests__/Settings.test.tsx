/**
 * @format
 */

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import SettingsScreen from '../Views/Settings';

it('renders correctly', () => {
  const tree = renderer.create(<SettingsScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
