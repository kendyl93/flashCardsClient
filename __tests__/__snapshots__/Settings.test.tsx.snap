// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "whitesmoke",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Settings Screen
  </Text>
</View>
`;
